ui/app/templates/components/namespace-link.hbs

```diff
@@ -1,5 +1,5 @@
 <a
-  href="{{href-to 'vault.cluster.secrets' (query-params namespace=this.normalizedNamespace)}}"
```

Comment:
We can remove the `href-to` dep, since this is the only place it was used

Reply:
good catch